src/main/java/pl/javastart/restoffers/offer/OfferService.java

```diff
@@ -0,0 +1,44 @@
+package pl.javastart.restoffers.offer;
+
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class OfferService {
+    private final OfferRepository offerRepository;
+    private final OfferDtoMapper offerDtoMapper;
+
+    public OfferService(OfferRepository offerRepository, OfferDtoMapper offerDtoMapper) {
+        this.offerRepository = offerRepository;
+        this.offerDtoMapper = offerDtoMapper;
+    }
+
+    public List<OfferDto> maptoDtos(List<Offer> offerList) {
+        return offerList.stream()
+                .map(offerDtoMapper::mapToDto)
+                .toList();
+    }
```

Comment:
Ta metoda może być prywatna.
Skoro wydzieliłaś jednak osobnego DtoMappera, to przeniósłbym tę metodę do niego